content/posts/deelnemers-netwerk.md

```diff
@@ -126,6 +126,11 @@ Zelf ook deelnemer worden? Lees [hier meer over de procedure](/aansluiten).
             <td>Herre Uittenhout</td>
             <td><a href="mailto:uittenhout@zorgdomein.nl">uittenhout@zorgdomein.nl</a></td>
         </tr>
+        <tr>
+            <td><a href="[https://www.zorginstituutnederland.nl/](https://www.zorginstituutnederland.nl/)/">Zorginstituut Nederland</a></td>
```

Comment:
Dit is geen valide html.
De href moet de directe URL bevatten, dus niet met die blokhaken:

`<a href="https://www.zorginstituutnederland.nl">Zorginstituut Nederland</a>`

Reply:
Fixed